MobilityData/gbfs-validator
# Feat/Github submodules for gbfs schemas

#186 
This PR adds [gbfs-json-schema](https://github.com/MobilityData/gbfs-json-schema) as a git submodule to assure that we have one source of truth regarding the gbfs schemas

What changed
- Existing schema files were deleted from the repository
- gbfs-json-schema added as a git submodule
- gbfs.js modified to reflect changes
- submodules update on `yarn install` making it easy for new developers to pick up
- github workflow updated to acomodate submodules

To test
Go on preview url https://deploy-preview-187--gbfs-validator.netlify.app/ and check functionality
